conftest.py

```diff
@@ -77,22 +76,23 @@ def keyring_reset():
 def generate_initial_config():
     """Generate an initial state for the configuration directory in tests.constants.ROOT_FS
     """
-    if not constants.ROOT_FS.exists("msui/testdata"):
-        constants.ROOT_FS.makedirs("msui/testdata")
+    _path = os.path.join(constants.ROOT_DIR, "msui", "testdata")
+    if not os.path.exists(_path):
+        os.makedirs(_path)
```

Comment:
constants.ROOT_DIR was for tests always a os path, see https://github.com/Open-MSS/MSS/blob/stable/tests/constants.py#L50

Reply:
OK, at the time I wrote the comment I was off the impressions that we wanted to use fs everywhere.

This can be simplified to
```python
(pathlib.Path(constants.ROOT_DIR) / "msui" / "testdata").mkdir(parents=True, exist_ok=True)
```
(maybe we should make the constants `pathlib.Path` objects instead of just strings at some point?)

Reply:
There are limitations by QT, qsettings needs an os.path. And for other parts we would need to change the filebrowser to the filepicker. 

After the release of 9.0.0 we can discuss and refactor that.  I think I close this PR and just fix the msui-core.conf mismatch.

Reply:
> There are limitations by QT, qsettings needs an os.path.

There is no such thing as "a os.path". os.path is not a type. Historically, os and os.path represented paths simply as strings, but the python stdlib should accept so-called PathLike objects anywhere now. Qt might not, but pathlib.Path can trivially be cast to str to get a string reprensentation that would work.

Reply:
It did not understand to handle an fs object.

Reply:
That was too fast typed. The failure comes because qsettings can't handle fs_urls or strings for pyfilesystem2.
Also the Qt filebrowser can't. 

For me it is ok after the release of 9.0.0 to look how we can refactor this. Also pathlib is nice. But now I want to get a  new major before the next campaign starts released.